### DISCUSSION TITLE
OpenSSF Scorecard

### DISCUSSION THREAD

**Discussion by sheldonhull** [aquaproj/aqua#2931] [Deprecated (Proposal)] [answered]
### Overview

```
docker run -e GITHUB_AUTH_TOKEN=$GITHUB_TOKEN gcr.io/openssf/scorecard:stable --repo=github.com/aquaproj/aqua
```

Can we get this running on the aqua projects? [opensssf scorecard](https://github.com/ossf/scorecard) and [scorecard site](https://scorecard.dev/) I include those on the renovate PR's, so nice to see it showing up. Aqua repo didn't do too bad either! 

### Why is the proposal needed?

Nice to have :-)

### Note

Take a peek at github action here: https://github.com/marketplace/actions/ossf-scorecard-action.
If you create a github-actions repo I'd contribute the reusable PR there. 

For example: https://github.com/sheldonhull/github-workflows/ and then i can submit the reusable workflow to your different repos. 

NOTE: This isn't an important/urgent request, just something that seems nice to have and I'm willing to contribute so it's minimal effort if you like the idea and have a github resuable workflow to submit the change to.

**Comment by suzuki-shunsuke** [comment]
Thank you for your suggestion.
I'm not familiar with OpenSSF Scorecard, but it looks nice.
I'll take a look.

**Reply by suzuki-shunsuke** [reply]
I tried OpenSSF Scorecard with another repository. https://github.com/suzuki-shunsuke/mkghtag

https://github.com/suzuki-shunsuke/mkghtag/security/code-scanning

<img width="996" alt="image" src="https://github.com/aquaproj/aqua/assets/13323303/7df7ae4d-1b27-4b05-a266-2017875da228">

- Add a workflow https://github.com/suzuki-shunsuke/mkghtag/pull/321
- Add a badge https://github.com/suzuki-shunsuke/mkghtag/pull/323

https://securityscorecards.dev/viewer/?uri=github.com/suzuki-shunsuke/mkghtag

<img width="325" alt="image" src="https://github.com/aquaproj/aqua/assets/13323303/825a75ff-c84a-4919-8281-607e054bceff">

**Reply by suzuki-shunsuke** [reply, marked as answer]
I added a workflow.

https://securityscorecards.dev/viewer/?uri=github.com/aquaproj/aqua

- https://github.com/aquaproj/aqua/pull/2933
  - https://github.com/aquaproj/aqua/blob/main/.github/workflows/push-main.yaml
  - https://github.com/suzuki-shunsuke/ossf-scorecard-workflow
